### PR TITLE
10 welcome cloud meter

### PR DESCRIPTION
**Pull request by cdcabrera**
# Welcome to Cloud Meter view

* Update account view with welcome copy
* constants for account, confirmationModal
* convenience scripts for local openshift install and run
* component layout for account wizard, confirmation modal
* redux layout for account view, account wizard, confirmation modal
* component and snapshots tests for account, wizard
* redux testing for account, wizard, confirmation, toasts

updates #10 

jun-14-2018 03-09-40

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 coverage:
   precision: 2
-  range: "60…100"
+  range: "60...100"
```
Comment (ilyavw) [inline .codecov.yml]: This defines just the range of color used. Try the config below:
```
coverage:
  precision: 2
  round: down
  range: "55...95"

  status:
    project:
      default:
        target: "60%"
    patch:
      default: off
    changes:
      default: off

comment:
  layout: "header, reach, diff, flags, files, footer"
  behavior: default
  require_changes: no
  require_base: no
  require_head: yes
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: .codecov.yml (hunk 1 above)
Comment (cdcabrera) [reply .codecov.yml]: Can do!

File: src/components/confirmationModal/confirmationModal.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <Modal.Footer>
+        <Button autoFocus bsStyle="default" className="btn-cancel" onClick={cancel}>
+          {cancelButtonText}
+        </Button>
+        <Button bsStyle="primary" onClick={onConfirm}>
+          {confirmButtonText}
+        </Button>
+      </Modal.Footer>
+    </Modal>
+  );
```
Comment (jeff-phillips-18) [inline src/components/confirmationModal/confirmationModal.js]: Keep an eye on https://www.github.com/patternfly/patternfly-react/pull/363

**Review by jeff-phillips-18 — DISMISSED** [review]
👍

**Review by ghost — APPROVED** [review]

**Comment by cdcabrera** [comment]
@cloudigrade/quality-engineering the convenience scripts were added to this pr to review them.
1. You'll need the base install requirements for Frontigrade, see ReadMe
2. Run the npm script `npm run start:review`
3. If you want to change the base url you'll need to create a `.env.local` file in the root of the project with a new line for env variable of `API_HOST=your-new-url`, then re-run the command `npm run start:review`

If it gets tedious to update `.env.local` give me a heads up and can approach it again.

**Review by elyezer — APPROVED** [review]
I was able to configure the UI to access my local cloudigrade deployment, thank you.

**Comment by cdcabrera** [comment]
@elyezer awesome, np!

**Pull request merged**